doc-content/enterprise-only/admin-and-config/dm-server-configure-proc.adoc

```diff
@@ -1,17 +1,17 @@
 [id='dm-server-configure-proc']
 = Configuring the {PRODUCT} server
 
-You can configure Red Hat JBoss Developer Studio to run the {PRODUCT} server.
+You can configure Red Hat Developer Studio to run the {PRODUCT} server.
```

Comment:
Don't worry about now but revisit after the release - you need to expand this description. So what if I can? Why would I want to ;-)

Reply:
Ok, I will keep this open to revisit.